repository/BaselineOfWidgets.package/BaselineOfWidgets.class/instance/baseline..st

```diff
@@ -45,4 +41,4 @@ baseline: spec
 			package: 'WidgetPharoPlatform' with: [
 				spec requires: #('Widgets')];
 			package: 'Widgets' with: [
-				spec includes: #('WidgetPharoPlatform')]].
```

Comment:
I haven't changed anything WidgetSqueakPlatform-related here, it's just the line ending that has changed.